community/settings.py

```diff
@@ -215,3 +214,9 @@
         'avatar': {'size': (200, 200), 'crop': True},
     },
 }
+
+#CELERY
+CELERY_BROKER_URL = os.environ.get('REDIS_URL')
```

Comment:
It can be! It's not how I'm using it _right now_ but it's certainly something we should look into doing for sql queries before we launch.  I'm using it for some asynchronous timed tasks in Meetups.  A task is scheduled to be executed at a specific time based on the duration property of the meetup.